index.js

```diff
@@ -149,23 +186,28 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+let current = 1
+function takeATicketNumber(line) {
+  line.push(current)
+  console.log(`Welcome. You are ticket number ${current}`)
+  current += 1
+}
```

Comment:
Nice work on this! Only issue is that the function is meant to **return** the string with the welcome message, not `console.log` it. You could re-write it like this:

```js
let current = 0
function takeATicketNumber(line) {
  current += 1
  line.push(current)
  return `Welcome. You are ticket number ${current}`
}
```